internal/model/game.go

```diff
@@ -1,24 +1,28 @@
 package model
 
+type Game struct {
```

Comment:
Did you merge two PRs and your logic into one PR? :D Are you sure this PR is ready to review? Maybe let's wait and merge this one as the last one.

https://github.com/HackYourCareer/SmartKickers/pull/64/files